scripts/hail_batch/eqtl_hail_batch/launch_round2.conditional_analysis.py

```diff
@@ -0,0 +1,63 @@
+"""Launch analysis runner for all cell types and chromosomes"""
```

Comment:
this script basically mirrors the former

Reply:
I think all the same feedback applies :)